main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+// Add the database path
+const DB string = "./data.csv"
+
+// Create pokemon struct
+type Pokemon struct {
+	Id int `json:"id"`
+	Name string `json:"name"`
+}
+
+// Global array for pokemons
+var Pokemons []Pokemon
+
+// readDB reads a CSV file and return the matrix of entries
+func readDB() [][]string{
+	log.Println("INFO - Reading DB")
+
+	var file, err = os.OpenFile(DB, os.O_RDWR, 0644)
+	if err != nil {
+		log.Fatalf("ERROR - %s", err.Error())
```

Comment:
I consider that this error should not be a `fatal` one, instead of this you can return it as part of the function returning values and manage it as HTTP status `Internal server error` with the error message. This will keep the server running and attending to any other requests and if the data file is fixed after, the service will automatically recover and displays de data without restarting the app